custom_components/tahoma/climate.py

```diff
@@ -1,30 +1,48 @@
 """Support for TaHoma climate devices."""
+import logging
+from pprint import pformat
 
 from homeassistant.components.climate import DOMAIN as CLIMATE
+from homeassistant.helpers import entity_platform
 
 from .climate_devices.atlantic_electrical_heater import AtlanticElectricalHeater
 from .climate_devices.dimmer_exterior_heating import DimmerExteriorHeating
 from .climate_devices.somfy_thermostat import SomfyThermostat
+from .climate_devices.stateless_exterior_heating import StatelessExteriorHeating
 from .const import DOMAIN
 
+_LOGGER = logging.getLogger(__name__)
+
 TYPE = {
     "AtlanticElectricalHeater": AtlanticElectricalHeater,
     "SomfyThermostat": SomfyThermostat,
     "DimmerExteriorHeating": DimmerExteriorHeating,
+    "StatelessExteriorHeating": StatelessExteriorHeating,
 }
 
+SERVICE_CLIMATE_MY_POSITION = "set_climate_my_position"
+SUPPORT_MY = pow(2, 20)
```

Comment:
Should we just default to the number value? I believe this function call is not necessary and doesn't improve readability. (since the other ones are specified as a number, thus I would need to now the result of pow(2, 20) to figure out if there are duplicates.